pkg/kudoctl/kudoinit/result.go

```diff
@@ -0,0 +1,98 @@
+package kudoinit
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/gosuri/uitable"
+)
+
+// Result holds the errors and warnings of a package verification
+type Result struct {
+	Errors   []string
+	Warnings []string
+}
+
+// NewResult initializes the Result struct
+func NewResult() Result {
```

Comment:
I'm completely against this style of error handling for different reasons depending on direction.
Issues with using this direction:
1. When do we use it compared to standard go error handling?  Guidance would need to be provided.    
2. It lives in the `kudoinit` package and is NOT kudoinit specific
3. It is a copy and paste of `verifier.result.go` with no added value.   We would want to collapse this into a shared lib

I'm strongly against this altogether...   

There is a need in the verification space to not "error out" on first error but to have a full report of all errors and warnings.

For init this is standard go code to provide a function...  if we want a run time "verification" than lets build that which would have a list of all conditions reported (and perhaps called within init) but lets not have it here like this...  It doesn't make sense.  This is NOT standard go error handling.   Unless we want to take this on for all code (which I don't).

TL; DR: We need to remove this collection of errors and warnings in init or switch to using a common set of similar code to verification

Reply:
Moved Results to a common struct